src/crawlers/lotteria.py

```
@@ -0,0 +1,251 @@
+            # 페이지 로드 대기 시간 단축
+            time.sleep(2)  # 5초 -> 2초로 단축
+
+            # 영양 정보 테이블이 로드될 때까지 대기 (타임아웃 단축)
+            try:
+                WebDriverWait(driver, 8).until(  # 15초 -> 8초로 단축
+                    EC.presence_of_element_located(
+                        (By.CSS_SELECTOR, "table.tbl-row-info")
+                    )
+                )
+                logger.info("Found nutrition table with correct selector")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)